xdsl/dialects/arm_neon.py

```diff
@@ -149,9 +149,9 @@ def __init__(
     ):
         self.arrangement = arrangement
         vectors: Sequence[VectorWithArrangement] = []
-        for reg in regs:
-            assert isinstance(reg.type, NEONRegisterType)
-            vectors.append(VectorWithArrangement(reg, self.arrangement))
+        for register in regs:
+            assert isinstance(register.type, NEONRegisterType)
+            vectors.append(VectorWithArrangement(register, self.arrangement))
```

Comment:
I needed to change this now that I'm importing ```reg```, as the variable name was shadowing the import